frontend_tests/node_tests/compose.js

```diff
@@ -112,58 +111,6 @@ function initialize_handlers({override}) {
     compose.initialize();
 }
 
-test_ui("test_wildcard_mention_allowed", () => {
```

Comment:
Moved this test to `node_tests/compose_validate.js` as all wildcard mentions right now are used by it. So, testing there will be better.